ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/vlan-interfaces.md

```diff
@@ -409,6 +412,15 @@ interface Vlan336
    ipv6 enable
    ipv6 address 2a00:3::1/64
 !
+interface Vlan339
+   description v6 nd options
+   ipv6 nd cache expire 250
+   ipv6 nd cache dynamic capacity 900
+   ipv6 nd cache refresh always
```

Comment:
This command shows me invalid on EOS 4.32. Can you please confirm it this is platform or EOS version dependent

Reply:
let me double check if this is something platform dependent, but at least for me it is fine in 4.32.1F version:

```
B1-BR-0001(config-if-Vl10)#ipv6 nd cache refresh ?
  always  Force refresh on cache expiry

B1-BR-0001(config-if-Vl10)#ipv6 nd cache refresh always ?
  <cr>

B1-BR-0001(config-if-Vl10)#sh vers
Arista cEOSLab
Hardware version:
Serial number: 7ABC871EEEA686DFEFD7E4119B37E515
Hardware MAC address: 001c.73f6.f48b
System MAC address: 001c.73f6.f48b

Software image version: 4.32.1F
```

which switch model did you test with?